src/components/Dialog/Dialog.st.css

```diff
@@ -5,26 +5,31 @@
 
 :import {
     -st-from: "../IconButton/IconButton.st.css";
-    -st-default: TPAIconButton;
+    -st-named: skin-line;
 }
 
 /*Defaults*/
 :vars {
-    DefaultCloseButtonColor: color-5;
+    DefaultWiredCloseButtonColor: color-5;
+    DefaultCloseButtonColor: #000000;
+    DefaultWiredBackgroundColor: color-1;
+    DefaultBackgroundColor: #ffffff;
 }
 
 /*Overrides*/
 :vars {
     CloseButtonColor: --overridable;
+    BackgroundColor: --overridable;
 }
 
 /*Fallbacks*/
 :vars {
-    CurrentCloseButtonColor: color(fallback(value(CloseButtonColor), value(DefaultCloseButtonColor)));
+    WiredCloseButtonColor: color(fallback(value(CloseButtonColor), value(DefaultWiredCloseButtonColor)));
+    WiredBackgroundColor: color(fallback(value(BackgroundColor), value(DefaultWiredBackgroundColor)));
 }
 
 .root {
-    -st-states: mobile, rtl;
+    -st-states: mobile, rtl, wired;
```

Comment:
As I see it, `wired` is simply another skin. so I wouldn't add it as a state, but rather as a class.
This will remove the `-st-extends: root` from `overrideStyleParams`

Reply:
So just to see I understand, 
Theoretically `mobile` and `rtl` can also be a "skin".
So maybe we don't want to use stylable states at all?
and just use classes in all our components?

I think that could be an easier solution to maintain.

However, then it immediately raises the question, then why do stylable states exist at all?
I seem to be confused about how stylable works 😕